src/hooks/useAnalytics.ts

```diff
@@ -150,14 +150,16 @@ export const useAnalytics = () => {
       );
     }
   }, [status]);
-
   // AnalyticsEvent.NavigatePage
   const location = useLocation();
-
+  const previousPathRef = useRef(location.pathname);
   useEffect(() => {
     // Ignore hashchange events from <iframe>s x_x
-    if (location.pathname.startsWith('/'))
-      track(AnalyticsEvents.NavigatePage({ path: location.pathname }));
+    if (location.pathname.startsWith('/')) {
+      const previousPath = previousPathRef.current;
+      track(AnalyticsEvents.NavigatePage({ path: location.pathname, previousPath }));
+      previousPathRef.current = location.pathname;
+    }
   }, [location]);
```

Comment:
should this be location.pathname ?

Reply:
hmm yeah that makes sense. 🤔 can fix later though so we don't have to re-approve this PR